.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 script:
 - bundle exec rails db:{create,migrate} RAILS_ENV=test
 - bundle exec rake db:schema:load
-- bundle exec rake csv_load:all
+- bundle exec rake db:test:prepare
```

Comment:
Ahh yes! We came across this exact script in our researching for the other endpoint and getting it to deploy. Let's see if this works and loads up all the csv data through Travis CI